packages/eslint-plugin/tests/rules/no-unused-vars/no-unused-vars.test.ts

```diff
@@ -766,7 +766,7 @@ export function foo() {
     `,
     // https://github.com/typescript-eslint/typescript-eslint/issues/5152
     `
-function foo<T>(value: T): T {
+export function foo<T>(value: T): T {
   return { value };
 }
 export type Foo<T> = typeof foo<T>;
```

Comment:
This change means that this PR is actually a breaking change, should we move it to the next major?

Reply:
Heh, "breaking" is a loose word. Per https://typescript-eslint.io/users/versioning/#eslint-plugin, _"Fixes incorrect behavior in a rule that may or may not introduce additional reports."_ is not a breaking change.

If we took on a new major for everything that changed rule reports, we'd be on ... well, quite a large number. 😄